money-core/src/main/scala/com/comcast/money/core/internal/MDCSupport.scala

```diff
@@ -38,10 +38,14 @@ class MDCSupport(enabled: Boolean = Money.Environment.enabled) {
   private val MoneyTraceKey = "moneyTrace"
   private val SpanNameKey = "spanName"
 
-  def setSpanMDC(spanId: Option[SpanId]): Unit = if (enabled) {
-    spanId match {
-      case Some(id) => MDC.put(MoneyTraceKey, MDCSupport.format(id))
-      case None => MDC.remove(MoneyTraceKey)
+  def setSpanInfoMDC(spanInfo: Option[SpanInfo]): Unit = if (enabled) {
```

Comment:
@HaloFour  I'm assuming this is the breaking change you are referring too?

I would say that anything in core/internal should not be being accessed by outside users. Not sure how we would enforce privacy here.

Reply:
Yes.  And I can keep the other methods around, mark them as deprecated if we feel that's appropriate.  To be honest I used the change to bring up the bigger issue of the `propogate` misspelling which impacts one method name as well as numerous parameter names.

Reply:
I've decided to back off of those changes for now.  There are numerous ways that I think the ergonomics of the API can be improved but that can probably wait until a major release.

Reply:
@HaloFour I thought about this again recently and realized I was totally wrong. We should just make this package private to Money i.e. `private[money]`.

I will add a issue to our stack to make everything in internal package private to Money unless @paulcleary or yourself have any objections?